src/components/ContentWidgets/WYSIWYG/WYSIWYGEditor.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import { Editor } from 'react-draft-wysiwyg';
+import { EditorState, convertToRaw } from "draft-js";
+import '../../../../node_modules/react-draft-wysiwyg/dist/react-draft-wysiwyg.css';
```

Comment:
you can instead `import 'react-draft-w...ysiwyg.css'` instead of `../..` all the way to node_modules.